network/serde/jsonrpc.go

```diff
@@ -4,7 +4,21 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/statechannels/go-nitro/protocols"
+	"github.com/statechannels/go-nitro/protocols/directdefund"
 	"github.com/statechannels/go-nitro/protocols/directfund"
+	"github.com/statechannels/go-nitro/protocols/virtualdefund"
+	"github.com/statechannels/go-nitro/protocols/virtualfund"
+)
+
+type RequestMethod string
+
+const (
```

Comment:
I moved this here to avoid a circular import. I haven't given a ton of thought to where this `const` should actually live.